src/core/QCheck.mli

```diff
@@ -397,6 +397,31 @@ module Gen : sig
       Example: [char_range 'a' 'z'] for all lower case ascii letters.
       @since 0.13 *)
 
+  val bytes_size : ?gen:char t -> int t -> bytes t
+  (** Builds a bytes generator from a (non-negative) size generator.
+      Accepts an optional character generator (the default is {!char}).
+      @since NEXT_RELEASE *)
+
+  val bytes : ?gen:char t -> bytes t
+  (** Builds a bytes generator. Bytes size is generated by {!nat}.
+      Accepts an optional character generator (the default is {!char}).
+      See also {!bytes_of} and {!bytes_readable} for versions without
+      optional parameters.
+      @since NEXT_RELEASE *)
+
+  val bytes_of : char t -> bytes t
+  (** Builds a bytes generator using the given character generator.
+      @since NEXT_RELEASE *)
+
+  val bytes_printable : bytes t
+  (** Generator using the {!printable} character generator.
+      @since NEXT_RELEASE *)
+
+  val small_bytes : ?gen:char t -> bytes t
```

Comment:
There's now `QCheck.bytes_small : bytes arbitrary` and `QCheck2.Gen.bytes_small` (yay!)
but no `QCheck.Gen.bytes_small`?